frontend/packages/console-dynamic-plugin-sdk/src/extensions/topology-details.ts

```diff
@@ -169,3 +166,5 @@ export type PodsAdapterDataType<E = K8sResourceCommon> = {
 export type NetworkAdapterType = {
   resource: K8sResourceCommon;
 };
+
+export type DetailsTabSectionCallback = ExtensionHook<React.ReactElement | undefined, GraphElement>;
```

Comment:
Can we have another type for the Hook result? Maybe this?

```suggestion
export type DetailsTabSectionCallback = ExtensionHook<React.ReactElement | undefined, GraphElement>;
export type DetailsTabSectionResult = ExtensionHookResult<React.ReactElement | undefined>;
```